lib/output.js

```diff
@@ -94,11 +94,15 @@ module.exports = {
 
     if (outputLevel === 2) {
       newline = true;
-      process.stdout.write(`${styles.debug(" (" + time + ")\n")}`);
+      if (time !== 'NaN sec') {
```

Comment:
this check should be for integer values and not for strings...
please move this few lines upper where the `time` value is calculated